experimental/examples/falling-balls-mpp/build.gradle.kts

```diff
@@ -165,6 +165,12 @@ compose.experimental {
                 device = IOSDevices.IPAD_MINI_6th_Gen
                 buildConfiguration = "Debug"
             }
+            connectedDevice("Device") {
+                //Usage: ./gradlew iosDeployDevice
+                buildConfiguration = "Debug"
+                teamId = "N462MKSJ7M"
```

Comment:
Better not have a team id here, and throw an exception in Gradle logic when its not defined asking for team id.

Reply:
Good Idea, I will move teamId to local.properties

Reply:
Done